src/sentry/performance_issues/detectors/http_overhead_detector.py

```diff
@@ -113,6 +113,16 @@ def _is_span_eligible(self, span: Span) -> bool:
 
         if not span_op or not span_op == "http.client" or not protocol_version == "1.1":
             return False
+
+        # Check if any spans have filtered URLs
+        event_spans = self._event.get("spans", [])
+        span_index = str(event_spans.index(span) if span in event_spans else -1)
+        meta = self._event.get("_meta", {}).get("spans", {})
+        if span_index in meta:
+            has_filtered_url = meta[span_index].get("data", {}).get("url", {})
```

Comment:
dont you need to check if the url has `Filtered` in it?

Reply:
there are multiple types of filters that can be applied here (ip address for example won't show "Filtered" but will have something else). this approach just checks if some filtering has occurred on the span's url (doesn't specify which one)

Reply:
so any time a span is in the meta its because its filtered?

Reply:
i can't think of a reason why span's url would show up in the meta if it wasn't being altered in a way where we want to exclude it from detection